certified-connectors/CiviQ/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2868 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CiviQ",
+    "description": "The current documentation explain's how the Civiq api is configured and gives and explanation for each endpoint.\nAPI provide secure data about consultations, surveys and submissions. \n\nTo connect a portal to the custom connector and use it in flow we need to create a connector by selecting it in the connectors list or by creating from custom connector list. \nWe will need to provide the host of the portal without protocol and API key. API key could be taken from API configuration page.",
+    "version": "1.2",
```

Comment:
@RicardoMartins1993 since this is a new connector, can you please change this to "1.0"